flink-shaded-asm-9/pom.xml

```diff
@@ -25,11 +25,11 @@ under the License.
     <parent>
         <groupId>org.apache.flink</groupId>
         <artifactId>flink-shaded</artifactId>
-        <version>17.0</version>
+        <version>18.0</version>
```

Comment:
Shouldn't we do an update to 19.0 here (because it's on master)? Similar to how it was done in 725c0caf3abe7982cbcd1f99174aa7d3d881b6b8. I guess, we missed doing it when 17.0 was released. :thinking:

Reply:
> I guess, we missed doing it when 17.0 was released. 🤔

I believe we didn't update `flink-shaded` for one of the Flink minor upgrades, because there was (at that moment) no reason to do the update. I don't think that we're following the strict versioning, more an indicator that there's no compatibility guarantees between `flink-shaded` version upgrades.

Reply:
You're right that there is no need to update the version in `master` if there's no change that's applied. But it would be still good to add this as part of the release documentation, wouldn't it? To ensure that we're not forgetting to update the version if a change is applied later on.

Reply:
Thanks for the response 
updated to 19.0
also updated confluence [1] section `Checklist to declare the process completed`

[1] https://cwiki.apache.org/confluence/display/FLINK/Creating+a+flink-shaded+release